rest-api/dao/participant_counts_over_time_service.py

```diff
@@ -118,6 +119,8 @@ def get_facets_sql(self, facets, table_prefix='ps'):
       'table_prefix': table_prefix, 'test_hpo_id': self.test_hpo_id}
     facets_sql += ' AND NOT ps.email LIKE "%(test_email_pattern)s"' % {
       'test_email_pattern': self.test_email_pattern}
+    facets_sql += ' AND ps.withdrawal_status = %(not_withdrawn)i' % {
+      'not_withdrawn': WithdrawalStatus.NOT_WITHDRAWN}
```

Comment:
I didn't mention this before, but at some point we probably will want to change this code so that instead of inlining the hpo ID, email pattern, and NOT_WITHDRAWN status into the SQL, we use bound parameters and pass the values in separately like we do below with start_date and end_date, and elsewhere in our code, e.g.:

https://github.com/all-of-us/raw-data-repository/blob/4f8e8ee014ee6d5f27988974fcf79cbb7744a323/rest-api/offline/public_metrics_export.py#L203

It doesn't matter that much honestly in this case, but it's consistent with everything else, and makes it harder for us to screw things up with improper escaping of values.

Reply:
Agreed; added as TODO.